js/notify/settings/configure-alert-position.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const electron = require('electron');
+const BrowserWindow = electron.BrowserWindow;
+const app = electron.app;
+const ipc = electron.ipcMain;
+const log = require('../../log.js');
+const logLevels = require('../../enums/logLevels.js');
+const notify = require('./../electron-notify');
+
+const { getConfigField, updateConfigField } = require('../../config');
+
+let configurationWindow;
+let screens;
+let notfPosition;
+let notfDisplay;
+
+let windowConfig = {
+    width: 460,
+    height: 360,
+    alwaysOnTop: true,
+    skipTaskbar: true,
```

Comment:
Would we not want this to show up in the taskbar just in case the user switches back to the main window for some reason?